doc/source/examples/dem/silo/silo.rst

```diff
@@ -29,14 +29,15 @@ Parameter file
 Mesh
 ~~~~~
 
-Contrary to previous examples, in this example, we use a mesh generated using Gmsh. The Gmsh (with an extension of .msh) file is located inside the example folder. ``check diamond cells`` enables searching for diamond-shaped boundary cells and adding them to particle-wall contact search cells.
+Contrary to previous examples, in this example, we use a mesh generated using Gmsh. The Gmsh (with an extension of .msh) file is located inside the example folder. We refine this mesh once by setting ``initial refinement=1`` to ensure that the cells are sufficiently small to enable an efficient contact detection. ``check diamond cells=true`` enables searching for diamond-shaped boundary cells. Diamond cells are cell which only share a line with the boundary and not a complete face. Such cells can occur in unstructured grids, but they are detrimental to the stability of DEM simualtions. Enabling this option adds these cells to the particle-wall contact search cells and is necessary to ensure stable collisions with the wall in the vicinity of diamond cells.
```

Comment:
```suggestion
Contrary to previous examples, in this example, we use a mesh generated using ``Gmsh <https://gmsh.info/>``_. The Gmsh file (extension .msh) is located inside the example folder. We refine this mesh once by setting ``initial refinement=1`` to ensure that the cells are sufficiently small to enable efficient contact detection. This is important because if we do not apply the refinement, some cells become diamond-shaped. Diamond cells are cells that only share a line with the boundary and not a complete face. ``check diamond cells=true`` enables searching for diamond-shaped boundary cells. Such cells can occur in unstructured grids, but they are detrimental to the stability of DEM simulations. Enabling this option adds these cells to the particle-wall contact search cells and is necessary to ensure stable collisions with the wall in the vicinity of diamond cells.
```